lib/utils.py

```diff
@@ -112,6 +110,23 @@ def json_dump(obj):
 
 def url_fails(url):
     """
+    If it is streaming
+    """
+    if urlparse(url).scheme in ('rtsp', 'rtmp'):
+        if arch in ('armv6l', 'armv7l'):
+            run_omxplayer = omxplayer(url, info=True, _err_to_out=True, _ok_code=[0, 1])
+            for line in run_omxplayer.split('\n'):
+                if 'Input #0' in line:
```

Comment:
Yeah, that's what I thought. Oh well, in that case, let's do it this way. Not idea, but it is what it is.